### PR TITLE
Kristian's completed Home Work 2 OOP2

### PR DESCRIPTION
**Pull request by KristianLindberg**
Kristian's completed Home Work 2 OOP2

### CODE REVIEW

**Comment by KristianLindberg** [comment]
Kristian's completed Oop ver. 3 exersice.

File: js/super_hero.js [hunk 1]
```diff
@@ -1 +1,21 @@
-// var SuperHero = ...
+var superHero = function(power){
```
Comment (afeld) [inline js/super_hero.js:1]: Convention is to use TitleCase for class names – see above.

File: js/super_hero.js [hunk 2]
```diff
@@ -1 +1,21 @@
...
+	this.power = power;
+	this.costume = "cape";
+	this.sideKick = "Tim Armstrong";
+};
+
+superHero.prototype.comic = function(){
+	console.log(this.costume);
+};
```
Comment (afeld) [inline js/super_hero.js:9]: Not sure that this method name is descriptive of what it's doing...

File: js/tests.js [hunk 3]
```diff
@@ -3,3 +3,10 @@ QUnit.test("hello test", function(assert) {
 });
 
 // ADD TESTS HERE
+QUnit.test("side kick test", function(assert){
+	assert.strictEqual(plane.sideKick, "Dead", "Planes side kick was dynamically changed to dead.")
+	});
+
+QUnit.test("costume test", function(assert){
+	assert.strictEqual(car.costume, "cape", "Checks that costume is cape.")
+	});
```
Comment (afeld) [inline js/tests.js:12]: Having the instance creation and method calls within the tests will help show what each are testing, and making sure that one test is less likely to affect the other.

**Pull request closed without merging**